contracts/seed/Seed.sol

```diff
@@ -36,7 +36,7 @@ contract Seed {
     uint256 public feeAmountRequired;     // Amount of seed required for fee
     uint256 public price;
     uint256 public startTime;
-    uint256 public endTime;
+    uint256 public endTime;               // end time set by project admin, last restort end time, if max not reached
```

Comment:
"// set by project admin, this is the last resort endTime to be applied when maximumReached has not been reached by then"

Reply:
I'll have to add in two lines, because of lint rules.